core/pl/util.pm.sdoc

```diff
@@ -90,3 +90,8 @@ sub wf {
   close $fh;
   $_[0];
 }
+
+sub testpath {
+  $_ =~ s/-\*/-0000\*/;
```

Comment:
Good points. To which `sr` are you referring? Streaming reduce? I'm not sure how that's relevant here, the point of the function is just to convert one of the output paths from `hdfs:///path../part-*` to `hdfs:///path/../part-0000*`, so I can test things out on hadoop without running over thousands of partfiles.